transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/specification/httpx/ExtendedHandshakeIT.java

```diff
@@ -0,0 +1,139 @@
+/**
```

Comment:
I would recommend adding client only tests as package private methods just for reference, as suggested in my comment on the other test